packages/react/package.json

```diff
@@ -11,7 +11,7 @@
     "build:ts": "tsc --b ./tsconfig.project.json --force",
     "build:ts:watch": "tsc --b ./tsconfig.project.json -w",
     "core:build": "yarn --cwd ../core run build",
-    "lint": "eslint \"*/**/*.{ts,tsx}\" --ignore-pattern \"dist\"",
+    "lint": "eslint \"*/**/*.{ts,tsx}\" --ignore-pattern \"dist\" --resolve-plugins-relative-to ../../node_modules",
```

Comment:
I needed to add this to tell eslint where to find the plugins that are installed in the root node_modules.